test/routes/src/server/routes/t_github.js

```diff
@@ -34,24 +40,26 @@ describe('The GitHub API endpoint', () => {
         nock.cleanAll();
       });
 
-      it('should successfully return', (done) => {
-        supertest(app)
-          .get('/github/snapcraft-yaml/anowner/aname')
-          .expect(200, done);
+      it('should satisfy all request mocks', async () => {
```

Comment:
I was wrong, it does tell:

`The Launchpad API endpoint list snaps route when LP returns error "after each" hook for "should satisfy all request mocks"`

So to `afterEach` it goes...